iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -4596,6 +4668,151 @@
 			};
 			name = Release;
 		};
+		ED097E7C2BB80C330006ED01 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
```

Comment:
These settings should come from the common, release and debug configs, like it's done in other configurations in this project.

Reply:
All settings are following the `common.xcconfig`. When I change this or other settings and save the file - they all will be automatically updated.
The fact that the property is set here with the concrete value - this is how xcconfig works:

<img width="904" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/7226c09c-cde4-433c-b2d6-33a77ba67007">